src/Aspire.Dashboard/Components/Controls/SpanDetails.razor.cs

```diff
@@ -33,4 +38,24 @@ private bool ApplyFilter(SpanPropertyViewModel vm)
         return vm.Name.Contains(_filter, StringComparison.CurrentCultureIgnoreCase) ||
             vm.Value?.Contains(_filter, StringComparison.CurrentCultureIgnoreCase) == true;
     }
+
+    protected override void OnParametersSet()
+    {
+        _contextAttributes =
+        [
+            new KeyValuePair<string, string>("Source", ViewModel.Span.Scope.ScopeName)
```

Comment:
is there a reason these are hardcoded?

Reply:
Are you asking why the names aren't in resource files for translation? None of the "known" property names in the details screens are translated because none of the properties that come from OTEL are translated.

Another example: https://github.com/dotnet/aspire/blob/7d5a03251a9e7e3099e87ee34c9499dde3095288/src/Aspire.Dashboard/Components/Controls/StructuredLogDetails.razor.cs#L53-L64